std_msgs/Header header

float32 roll	# roll angle [deg]
float32 pitch	# pitch angle [deg]
float32 yaw	# yaw rate [deg/s]
float32 gaz	# vertical velocity [m/s]
